openflights.css

```diff
@@ -207,6 +207,14 @@ td.donate2 {
 }
 
 /* Autocomplete styles */
+.autocomplete {
+ /* TODO: improve this in code review
```

Comment:
@reedy Is there a better way to do this?

Reply:
I'm not sure, but I also don't see a problem with this either...

You're working around a UI bug by setting a display order (of sorts), so setting it high it should show on top of basically everything...

Reply:
It's also a temporary popup, not some UI element that is always on display